### PR TITLE
Increase parallelization so that jobs can run faster on SAGA

### PR DESCRIPTION
**Pull request by joecummings**
- Add key value management functions to ClaimDataset class
- Modify all jobs to run on ZipKeyValueStores instead of dirs
- Update unify_entities
- Modify params and rest of drivers to use key value stores

### CODE REVIEW

**Comment by joecummings** [comment]
TODO: Add in all the required dependencies in order to run this (includes pegasus wrapper, saga tools, vistautils, etc.)

File: cdse_covid/semantic_extraction/run_srl.py [hunk 1]
```diff
@@ -59,6 +59,6 @@ def main(inputs: Path, output: Path, *, spacy_model: Language) -> None:
     parser.add_argument("--spacy-model", type=Path)
```
Comment (elizlee) [inline cdse_covid/semantic_extraction/run_srl.py:59]: We don't need this anymore, then?

File: cdse_covid/pegasus_pipeline/claim_pipeline.py [hunk 2]
```diff
@@ -98,94 +91,121 @@ def main(params: Parameters) -> None:
...
+        output_locator = kvs.locator / "srl"
+        srl_python_file = srl_params.existing_file("python_file")
+        srl_output_dir = directory_for(output_locator) / "documents.zip"
+        srl_job = run_python_on_args(
+            output_locator,
+            srl_python_file,
+            f"""
+            --input {kvs.path} \
+            --output {srl_output_dir} \
+            --spacy-model {model_path} \
```
Comment (elizlee) [inline cdse_covid/pegasus_pipeline/claim_pipeline.py:141]: Looks like the `spacy-model` param can be removed

**Review by elizlee — APPROVED** [review]
Just a few nits

File: cdse_covid/pegasus_pipeline/claim_pipeline.py (hunk 2 above)
Comment (joecummings) [reply cdse_covid/pegasus_pipeline/claim_pipeline.py:141]: Yeah, for the sake of speed (of iteration), we are not going to be saving and reusing the same SpaCy model.

**Pull request merged**